wrappers/python/tests/layer/test_llama_attention.py

```diff
@@ -134,9 +144,10 @@ def generate_inputs(dtype: str, params: LlamaAttentionTestParams, bias: bool):
 ])
 class TestLlamaAttention:
 
-    def test_torch_coersion(self, starpu_simple, torch_rng, dtype: str,
+    def test_torch_coercion(self, starpu_simple, torch_rng, dtype: str,
                             params: LlamaAttentionTestParams, bias: bool):
-        torch_layer, nntile_layer, _, _ = generate_inputs(dtype, params, bias)
+        torch_layer, nntile_layer, _, _, _ = \
+                generate_inputs(dtype, params, bias)
```

Comment:
FYI You can use matching to variable length dummy arguments instead of enumerating dummy variable per element of result tuple as follows.

```python
torch_layer, nntile_layer, *_ = \
    generate_inputs(dtype, params, bias)
```

Please check you indentation configuration. It seems that indent width (8 spaces) is twice standard indent (4 spaces).